Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml

```diff
@@ -159,6 +159,33 @@ script:
     - contextPath: TAXII.Indicator.Type
       description: The indicator type.
       type: String
+    - contextPath: TAXII.Indicator.title
+      description: The indicator title.
+      type: String
+    - contextPath: TAXII.Indicator.description
+      description: The observable description.
+      type: String
+    - contextPath: TAXII.Indicator.stixindicatordescription
+      description: The indicator description.
+      type: String
+    - contextPath: TAXII.Indicator.stixindicatorname
+      description: The indicator title.
```

Comment:
What about the title description? Please make sure you don't have the same descriptions to different paths